### ISSUE TITLE
private final SOME_EXCEPTION

### ISSUE THREAD

**Issue by Warheart666** [savely03/booking_app#16]
на примере BookingService поясни, по каким причинам решил сделать константы исключений прям в сервисном классе?
А если этот сервис может выбрасывать 150 различных исключений, будет 150 констант?

**Comment by savely03** [comment]
Да, тут немного перемудрил (в этом сервисе просто всего 2 исключения и решил их сразу заинжектить), лучше стандартно выкидывать исключения (исправлю). Единственное у меня в CrudService есть константа исключения, которое является абстракцией, чтобы определить общее поведение для реализаций данного сервиса.
